lib/actions/FrontendAction.js

```diff
@@ -26,24 +26,30 @@ class FrontendAction {
 
   /**
    * Registers the frontend command.
-   * @param {Command} commander Instance of the commander module.
+   * @param {Command} caporal Instance of the commander module.
    */
-  register (commander) {
-    commander
-      .command('frontend <action>')
-      .description('The frontend action to run. Possible actions are \'start\' and \'setup\'.')
+  register (caporal) {
+    caporal
+      .command('frontend start')
+      .description('Start the webpack for the frontend development')
```

Comment:
Still missleading. It startS the webpack development server. From my understanding "starting webpack" implies that the source will be made production ready.